### PR TITLE
Update part-8-rtk-query-advanced.md

### PR DESCRIPTION
**Pull request by mohammadyousefvand**
Thanks for the PR!

To better assist you, please select the type of PR you want to create.

Click the "Preview" tab above, and click on the link for the PR type:

- [:bug: Bug fix or new feature](?template=bugfix.md)
- [:memo: Documentation Fix](?template=documentation-edit.md)
- [:book: New/Updated Documentation Content](?template=documentation-new.md)

### CODE REVIEW

File: docs/tutorials/essentials/part-8-rtk-query-advanced.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ export const EditPostForm = ({ match }) => {
     if (title && content) {
       // highlight-next-line
       await updatePost({ id: postId, title, content })
-      history.push(`/posts/${postId}`)
+      navigate(`/posts/${postId}`);
```
Comment (timdorr) [inline docs/tutorials/essentials/part-8-rtk-query-advanced.md:119]:
```suggestion
      navigate(`/posts/${postId}`)
```

We use `nosemi`

**Comment by timdorr** [comment]
Thanks!

**Comment by markerikson** [comment]
Hmm. For the record, the current version of the "Essentials" example app is still using React-Router v5.  So, technically I think this change to the code snippets in the docs is incorrect on two levels - it doesn't match the actual code in the sandbox / repo, and it is wrong for the version of React-Router we're using.

**Comment by timdorr** [comment]
Ah, gotcha. I've got repo whiplash from just working in the RR one 😀

**Pull request merged**